regional-pizza.md

```diff
@@ -1,43 +1,62 @@
-* Cold Toppings: After the crust and sauce are baked, cold toppings like cheese, pepperoni, and other desired ingredients are added on top. The heat from the crust slightly melts the cheese, but the toppings largely remain cold or at room temperature.
-* Rectangular Shape: Typically rectangular in shape and often sold by the slice.
-* Crispy Crust: The crust is usually chewy and crispy, providing a nice contrast to the cold toppings.
-Served Immediately: Due to its unique preparation method, Ohio Valley pizza is best enjoyed immediately after it's made to experience the contrast of hot crust and cold toppings.
+- Baked Crust First: Unlike traditional pizzas where toppings are added before baking, Ohio Valley pizza is first baked with just the crust and tomato sauce.
+- Cold Toppings: After the crust and sauce are baked, cold toppings like cheese, pepperoni, and other desired ingredients are added on top. The heat from the crust slightly melts the cheese, but the toppings largely remain cold or at room temperature.
+- Rectangular Shape: Typically rectangular in shape and often sold by the slice.
+- Crispy Crust: The crust is usually chewy and crispy, providing a nice contrast to the cold toppings.
+  Served Immediately: Due to its unique preparation method, Ohio Valley pizza is best enjoyed immediately after it's made to experience the contrast of hot crust and cold toppings.
 
 This style of pizza is beloved by many in the Ohio Valley region and has a dedicated following. It offers a different pizza-eating experience, with the juxtaposition of hot and cold elements in every bite.
 
 ## New York City
 
 New York City has had a major influence on the perception of pizza for decades. Here are the key characteristics of this iconic style of pizza:
 
-* Hand-tossed crust: This thin, crispy crust is made with high-gluten flour and stretched by hand to create a soft, bubbly interior.
-* Sweet sauce: The sauce is typically made with canned tomatoes, olive oil, garlic, sugar, and herbs. The sugar helps to balance out the acidity of the tomatoes.
-* Foldable slices: New York-style pizza is typically cut into eight large, triangular slices that are easy to eat on the go.
-Coal-fired oven: New York-style pizza is traditionally cooked in a coal-fired oven, but gas ovens are also commonly used.
+- Hand-tossed crust: This thin, crispy crust is made with high-gluten flour and stretched by hand to create a soft, bubbly interior.
+- Sweet sauce: The sauce is typically made with canned tomatoes, olive oil, garlic, sugar, and herbs. The sugar helps to balance out the acidity of the tomatoes.
+- Foldable slices: New York-style pizza is typically cut into eight large, triangular slices that are easy to eat on the go.
+  Coal-fired oven: New York-style pizza is traditionally cooked in a coal-fired oven, but gas ovens are also commonly used.
```

Comment:
Please refer to [my comment in the next line](https://github.com/open-sauced/pizza-verse/pull/44/files#r1366235972) for context on how to improve this section.
```suggestion
- Hand-tossed crust: This thin, crispy crust is made with high-gluten flour and stretched by hand to create a soft, bubbly interior.
- Sweet sauce: The sauce is typically made with canned tomatoes, olive oil, garlic, sugar, and herbs. The sugar helps to balance out the acidity of the tomatoes.
- Foldable slices: New York-style pizza is typically cut into eight large, triangular slices that are easy to eat on the go.
  Coal-fired oven: New York-style pizza is traditionally cooked in a coal-fired oven, but gas ovens are also commonly used.
```